src/components/charts/CandleChart.vue

```diff
@@ -91,7 +80,23 @@ const props = defineProps({
     type: Object as () => ChartSliderPosition,
     default: () => undefined,
   },
+  colorPreference: { required: false, type: String, default: 'greenUp' },
 });
+
+// Candle Colors
+const colorProfit = props.colorPreference === 'greenUp' ? '#26A69A' : '#EF5350';
+const colorLoss = props.colorPreference === 'greenUp' ? '#EF5350' : '#26A69A';
```

Comment:
Why is this using hardcoded colors, instead of the colorStore?

Reply:
I found that the color of candles (`#26A69A` and `#EF5350`) are different from the ones in  `colorProfit: '#12bb7b', colorLoss: '#ef5350'`, so I did not use the color in store directly

Reply:
That's correct - it should be different colors, really. using the profit/loss colors doesn't play well within the actual chart.